TTS/server/server.py

```diff
@@ -39,6 +39,7 @@ def create_argparser() -> argparse.ArgumentParser:
         help="Name of one of the pre-trained tts models in format <language>/<dataset>/<model_name>",
     )
     parser.add_argument("--vocoder_name", type=str, default=None, help="name of one of the released vocoder models.")
+    parser.add_argument("--speaker_name", type=str, default=None, help="name of the speaker to use in multi-speaker models.")
```

Comment:
For consistency with Coqui's normal CLI, I'd call this argument `--speaker_idx`:
https://github.com/idiap/coqui-ai-TTS/blob/6d476bcf9d2b2d05d39b138593a1a8bd95d12560/TTS/bin/synthesize.py#L219-L222

Reply:
This should be resolved